sound/soc/sdw_utils/soc_sdw_utils.c

```diff
@@ -363,6 +363,8 @@ struct asoc_sdw_codec_info codec_info_list[] = {
 				.num_controls = ARRAY_SIZE(generic_spk_controls),
 				.widgets = generic_spk_widgets,
 				.num_widgets = ARRAY_SIZE(generic_spk_widgets),
+				.quirk = SOC_SDW_CODEC_SPKR,
+				.quirk_exclude = true,
```

Comment:
The quirk is added to the rt721 dai. Is it correct?

Reply:
> The quirk is added to the rt721 dai. Is it correct?

Oh, thanks for pointing out, resolved.